OpenSim/Sandbox/ListInput.h

```diff
@@ -12,13 +12,13 @@ Input<T> updInput_##name(); \
 OutputValuesIterator getInputValues_##name() const; \
 
 
-#define OpenSim_DECLARE_CONNECTOR(name, T) \
-int _connector_##name { constructConnector<T>(name) }; \
+#define OpenSim_DECLARE_SOCKET(name, T) \
```

Comment:
Because this file resides in the Sandbox, I think it would be fine to leave it as it is—but it's also okay to change (e.g., to prevent confusion in the future).